templates/fetchers/edit.html.twig

```diff
@@ -0,0 +1,65 @@
+{% extends '@EasyAdmin/default/layout.html.twig' %}
+
+{% form_theme form with easyadmin_config('design.form_theme') only %}
+
+{% block content_header %}
+    <div class="d-flex flex-row justify-content-between align-content-center w-100">
+        <div class="content-header-title">
+            <h1 class="title">Edit Fetcher</h1>
+        </div>
+    </div>
+{% endblock %}
+
+{% block content_title %}
+    Edit {{ fetcher.fetcherName }}
+{% endblock %}
+
+{% block main %}
+
+    {{ form_start(form, { 'attr': {'class': 'edit-form'} }) }}
+
+    <fieldset>
+        <legend>
+            <i class="fa fa-fw fa-pencil"></i>
+            {{ fetcher.fetcherName }}
+        </legend>
+        <div class="row">
+            <div class="col-12">
+                {{ form_row(form.tags) }}
+            </div>
+            {% if form.allowEmptyTags is defined %}
+                <div class="col-12">
+                    {{ form_row(form.allowEmptyTags) }}
+                </div>
+            {% endif %}
```

Comment:
Nope :/ 
Nothing spefic is allowed here.
IMHO, the best way is to call form_form(form.configuration).

And maybe we will need to nest the form (as said few days ago).
It'll be cleaner and avoir the "hack" in the admin form `unset($data['active])`.